spec.md

```diff
@@ -8,66 +8,96 @@ title: Semantic Specification
 
 
 ~~~
-Parentage/causal relationships for spans from a single trace
+Causal relationships for Spans in a single trace
 
-        [Span A]  <--(the root span)
+        [Span A]  ←←←(the root span)
             |
      +------+------+
      |             |
- [Span B]      [Span C]
+ [Span B]      [Span C] ←←←(Span A is Span C's parent)
      |             |
- [Span D]      +---+-----+
-               |         |
-           [Span E]  [Span F]
-                         |
-                +--------+--------+
-                |        |        |
-            [Span G] [Span H] [Span I]
+ [Span D]      +---+-------+
+               |           |
+           [Span E]    [Span F] >>> [Span G] >>> [Span H]
```

Comment:
yes, it could fan out... in my experience that's been less common than having each step in the sequence be its own "mini-root-span" where it acts as a `Parent` to any children that execute in parallel. I thought it best not to introduce that level of complexity to the diagram, though.